k8s/demo/common/bsp/bulk-scan-processor.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: bulk-scan-processor
+  namespace: bsp
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bulk-scan-processor
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bulk-scan-processor
+    version: 0.2.19
+  values:
+    java:
+      replicas: 1
```

Comment:
any reason to have just one replica ?

Reply:
mistake, fixed

Reply:
Default capacity we've got in processor for ASE is 1 and it's only different in aat and prod. I guess that's where it comes from. Other configs for AKS demo have 2, so it make sense to stick with 2.